js/notify/electron-notify.js

```diff
@@ -99,28 +98,37 @@ let config = {
         color: '#CCC'
     },
     defaultStyleTitle: {
-        fontFamily: 'Arial',
+        fontFamily: 'Lato-Bold, san-serif',
         fontSize: 14,
         fontWeight: 700,
-        opacity: 0.87,
-        marginRight: 10,
-        alignSelf: 'center',
+        color: '#4a4a4a',
+        marginLeft: 52,
+        overflow: 'hidden',
+        display: '-webkit-box',
+        webkitLineClamp: 1,
+        webkitBoxOrient: 'vertical',
+    },
+    defaultStylePod: {
+        fontFamily: 'Lato-Regular, san-serif',
```

Comment:
Same as above. Also, I wonder why we should use Lato here. Are we including the font as part of the package or in any other way? If not, it's not really useful as Lato is never almost included as part of OS fonts.